specification/sql/resource-manager/readme.md

```diff
@@ -492,6 +492,7 @@ input-file:
  - ./Microsoft.Sql/preview/2019-06-01-preview/serverOperations.json
  - ./Microsoft.Sql/preview/2019-06-01-preview/WorkloadGroups.json
  - ./Microsoft.Sql/preview/2019-06-01-preview/WorkloadClassifiers.json
+ - ./Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json
```

Comment:
Update other language's readme.*.md?

Reply:
Sorry I don't understand, can you explain?

Reply:
I think Allen is remind you have a check whether there is other change also for other readme file for specific language which in same folder of readme.md. if no, it is fine